pkg/operator/ceph/csi/spec.go

```diff
@@ -419,14 +419,6 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 		})
 	}
 
-	holderEnabled = !CSIParam.EnableCSIHostNetwork
-
-	for i := range r.clustersWithHolder {
-		if r.clustersWithHolder[i].cluster.Spec.Network.IsMultus() {
-			holderEnabled = true
-			break
-		}
-	}
```

Comment:
This logic has moved up into the parent `controller.go` logic here: https://github.com/rook/rook/pull/13890/files#r1515280100